### PR TITLE
Adds multi-arch image support to trust

### PR DESCRIPTION
**Pull request by JoshVanL**
Signed-off-by: joshvanl <vleeuwenjoshua@gmail.com>
/assign @irbekrm 

After merging we can release v0.1.0

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,11 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/distroless/static@sha256:aadea1b1f16af043a34491eec481d0132479382096ea34f608087b4bef3634be
-LABEL description="cert-manager operator for distributing trust bundles across a Kubernetes cluster"
+# Build the trust binary
+FROM docker.io/library/golang:1.17 as builder
```
Comment (irbekrm) [inline Dockerfile:16]: go.mod file still has Go 1.16, maybe you want to change that in this PR as well?

**Review by irbekrm — APPROVED** [review]
/lgtm
/hold

Holding in case you want to bump Go version in go.mod

File: Dockerfile (hunk 1 above)
Comment (JoshVanL) [reply Dockerfile:16]: Thanks, done!

**Pull request merged**